sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -454,6 +454,7 @@
 
   /**
    * @since 4.0
+   * @deprecated since 5.2
    */
   String ANALYSIS_MODE_INCREMENTAL = "incremental";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @Deprecated annotation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)